packages/connectivity_plus/pubspec.yaml

```diff
@@ -24,23 +28,18 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
-  connectivity_plus_platform_interface: ^0.4.1
-  connectivity_plus_linux: ^0.3.1
-  connectivity_plus_macos: ^0.4.1
-  connectivity_plus_web: ^0.6.1
-  connectivity_plus_windows: ^0.1.1
+  meta: ^1.3.0
+  connectivity_plus_platform_interface: ^0.5.0-nullsafety.0
+  connectivity_plus_linux: ^0.4.0-nullsafety.0
+  connectivity_plus_macos: ^0.5.0-nullsafety.0
+  connectivity_plus_web: ^0.7.0-nullsafety.0
+  connectivity_plus_windows: ^0.2.0-nullsafety.0
```

Comment:
one more: could you please change `^x.y.z-nullsafety.N` to `^x.y.z-nullsafety` to make it more flexible (helps the integration test to pass once `.1` is out)

Reply:
sorry it keeps going on going, but we're almost there! :smile:  this was a pretty painful package to migrate

Reply:
no issue! 😃I should thank you, for succesfully investigating and reproducing all of the encountered errors 👏👍

Reply:
dropping the .0 suffix from these connectivity_plus_xxx dependencies here should be enough so we don't need to bump them after each fix :)